src/app/rosetta/docker-test-start.sh

```diff
@@ -51,7 +51,7 @@ MINA_CONFIG_FILE="${MINA_CONFIG_DIR}/daemon.json}"
 export PK=${PK:-"B62qiZfzW27eavtPrnF6DeDSAKEjXuGFdkouC3T5STRa6rrYLiDUP2p"}
 SNARK_PK=${SNARK_PK:-"B62qjnkjj3zDxhEfxbn1qZhUawVeLsUr2GCzEz8m1MDztiBouNsiMUL"}
 
-CONFIG_TEMPLATE=${CONFIG_TEMPLATE:-/genesis_ledgers/daemon.json.template}
+CONFIG_TEMPLATE=${CONFIG_TEMPLATE:-./genesis_ledgers/daemon.json.template}
```

Comment:
Remove this, the docker- scripts are all meant to be run in the container so at least the defaults should target docker